client/src/modules/stock/exit/exit.js

```diff
@@ -575,10 +575,18 @@ function StockExitController(
         documentUuid = document.uuid;
 
         // update requisition status if needed
-        if (!vm.requisition) { return null; }
-
-        const COMPLETED_STATUS = 2;
-        return Stock.stockRequisition.update(vm.requisition.uuid, { status_id : COMPLETED_STATUS });
+        if (!vm.requisition.uuid) { return null; }
+
+        const movementRequisition = {
+          stock_requisition_uuid : vm.requisition.uuid,
+          document_uuid : documentUuid,
+        };
+
+        const COMPLETED_STATUS = 6;
```

Comment:
Is there somewhere global that this can be defined?  Like `bhConstant` or something like that?  I'm not sure I understand why it changed from "2" to "6" since both numbers don't mean anything to me in this context.